openlibrary/macros/databarWork.html

```diff
@@ -63,10 +53,10 @@
 
 $ library = None
 <style type="text/css">
-#edit-identifiers { 
+#edit-identifiers {
                     font-size: 75%;
```

Comment:
(a) is there a reason the css indentation is inconsistent, and (b) is there a reason these styles are not imported from a stylesheet?